crates/bevy_math/src/curve/easing.rs

```diff
@@ -296,3 +297,121 @@ impl ElasticCurve {
         Self { omega }
     }
 }
+
+/// Curve functions over the [unit interval], commonly used for easing transitions.
+///
+/// [unit interval]: `Interval::UNIT`
+#[derive(Debug, Copy, Clone, PartialEq)]
```

Comment:
I would really like this to be Reflect, so you can twiddle this in inspectors.

Reply:
added serialisation while I'm at it!